patches/minecraft/net/minecraft/tileentity/TileEntityChest.java.patch

```diff
@@ -0,0 +1,26 @@
+--- ../src-base/minecraft/net/minecraft/tileentity/TileEntityChest.java
++++ ../src-work/minecraft/net/minecraft/tileentity/TileEntityChest.java
+@@ -468,4 +468,23 @@
+             this.field_145985_p[i] = null;
+         }
+     }
++
++    @Override
++    public <T> T getCapability(net.minecraftforge.common.capabilities.Capability<T> capability, net.minecraft.util.EnumFacing facing)
++    {
++        if (capability == net.minecraftforge.items.CapabilityItemHandler.ITEM_HANDLER_CAPABILITY)
++        {
++            net.minecraftforge.items.IItemHandler chestWrapper = net.minecraftforge.items.wrapper.VanillaWrapperHandler
++                    .getChestWrapper(this, field_174879_c, field_145850_b);
++            if (chestWrapper != null)
```

Comment:
This creates a new entry every query. TEs/Blocks have a neighbor update event it'd probably be better to detect the double chest there and setup the wrapper. That way the query is as fast as possible.